src/controllers/screening-grade.ts

```diff
@@ -1,55 +1,55 @@
 import { Request, Response } from 'express';
+import mongoose from 'mongoose';
 import Applicant from '../models/applicant';
 import ScreeningGrade, { IScreeningGrade } from '../models/screening-grade';
 
 export const createScreeningGrade = async (req: Request<any>, res: Response): Promise<void> => {
+    const allowedLevel = ['Beginner', 'Independent', 'Experienced'];
+    const allowedScore = [0, 1, 2];
```

Comment:
Response body no longer does the c4, c5, c6 business. It just takes the level of the applicant and directly inserts it into the applicant model